src/jats/importer/jats-body-transformations.ts

```diff
@@ -415,4 +417,91 @@ export const jatsBodyTransformations = {
       body.prepend(section)
     }
   },
+  moveReferencesToBackmatter(
+    body: Element,
+    references: References | undefined,
+    createElement: (tagName: string) => HTMLElement
+  ) {
+    const backmatter = body.querySelector('sec[sec-type="backmatter"]')
+
+    if (!references || !references.items.size || !backmatter) {
+      return
+    }
+
+    const bibliographySection = this.createBibliographySection(createElement)
+    const bibliographyElement = createElement('bibliography-element')
+
+    references.getBibliographyItems().forEach((item) => {
+      const bibliographyItem = this.createBibliographyItem(item, createElement)
+      bibliographyElement.appendChild(bibliographyItem)
+    })
+
+    bibliographySection.appendChild(bibliographyElement)
+    backmatter.appendChild(bibliographySection)
+  },
+  createBibliographySection(createElement: (tagName: string) => HTMLElement) {
+    const section = createElement('sec')
+    section.setAttribute('sec-type', 'bibliography_section')
```

Comment:
We should change this to `bibliography`, otherwise it won't be detected correctly.

Reply:
Good catch, thanks, i'll update it.